packages/api/cli/spec/cli.spec.ts

```diff
@@ -0,0 +1,19 @@
+import path from 'node:path';
+
+import { spawn } from '@malept/cross-spawn-promise';
+import { describe, expect, it } from 'vitest';
+
+function runForgeCLI(...extraArgs: string[]): Promise<string> {
+  const args = ['ts-node', path.resolve(__dirname, '../src/electron-forge.ts'), ...extraArgs];
+  return spawn('npx', args);
+}
+
+describe('cli', { timeout: 30_000 }, () => {
```

Comment:
how was 30s determined for this test? I'm wondering if we can revisit this and adjust as necessary

Reply:
I had the default and kept increasing it until it stopped flaking 😓

I think a future task would maybe be if we can move this to the slow test suite as well.